
Invalidate trace cache @ step 1: expected module 1, but got module 0
/usr/local/lib/python3.10/dist-packages/pytorch_lightning/trainer/call.py:54: Detected KeyboardInterrupt, attempting graceful shutdown...